### PR TITLE
Compute pythonVersion dynamically

### PR DESCRIPTION
**Pull request by paulproteus**
This allows the same template to be used with any Python version. Computing the Python version takes between 0ms and 1ms on an Android emulator on my Mac Mini.

If we merge this, we can then immediately push the `dev` branch to `3.6` and `3.7` and `3.8`.

## Testing

I validated that an app works with a Python 3.6 or 3.7 or 3.8 virtualenv, using this template (no change to the template when switching Python versions). I did adjust the `support_package` to point to a local build of all those Python versions.

### CODE REVIEW

File: {{ cookiecutter.formal_name }}/app/src/main/java/org/beeware/android/MainActivity.java [hunk 1]
```diff
@@ -158,8 +159,25 @@ private void startPython(Map<String, File> paths) throws Exception {
...
+                    @Override
+                    public boolean accept(File file, String s) {
+                        return s.startsWith("libpython") && s.endsWith(".so");
+                    }
+                }
+        );
+        if (libpythons.length == 0) {
+            throw new Exception("Unable to compute Python version");
+        }
+        String pythonVersion = libpythons[0].replace("libpython", "").replaceAll("m*.so", "");
```
Comment (freakboy3742) [inline {{ cookiecutter.formal_name }}/app/src/main/java/org/beeware/android/MainActivity.java:176]: Will this work for Python3.8, which doesn't have the `m` suffix?

Comment (paulproteus) [reply {{ cookiecutter.formal_name }}/app/src/main/java/org/beeware/android/MainActivity.java:176]: Yup -- it's a regex, so the `m*` permits the absence of `m`. I tested with all 3 versions.

**Comment by paulproteus** [comment]
> Makes sense; one potential problem flagged inline; and did you actually confirm the startup time impact? I'm guessing it won't be much, but just to be sure...

Yup, tested and it's been 0ms and 1ms on my Mac Mini's Android emulator.

File: {{ cookiecutter.formal_name }}/app/src/main/java/org/beeware/android/MainActivity.java (hunk 1 above)
Comment (freakboy3742) [reply {{ cookiecutter.formal_name }}/app/src/main/java/org/beeware/android/MainActivity.java:176]: Ah - right - I thought it was a glob.

**Pull request merged**